### PR TITLE
Call transformers for OD

### LINKED ISSUE #144: Call transformers for std hardware: od_sensor

### PR DESCRIPTION
**Pull request by jamienoss**
Resolves #144

### CODE REVIEW

**Comment by jamienoss** [comment]
@jamienoss refactor linearfit, etc, ``coefficients`` -> ``parameters`` to make uniform with other transformers.

File: evolver/hardware/standard/od_sensor.py [hunk 1]
```diff
@@ -29,5 +32,26 @@ def read(self):
...
+                # Calibrate raw data to density.
+                raw = int(raw)
+                density = self._transform("output_transformer", "convert_to", raw, vial)
+                self.outputs[vial] = self.Output(vial=vial, raw=raw, density=density)
         return self.outputs
+
+
+class OD90(ODSensor):
+    class Config(ODSensor.Config):
+        name: str = ODSensor.Config.model_fields["name"].from_field(default="od_90")
```
Comment (amitschang) [inline evolver/hardware/standard/od_sensor.py:44]: I'm not sure this is correct - I think the intention is to combine the `Field` object information, essentially overriding the default? but this looks like it doesn't do that (`from_field` is a static method)

```py
f = pydantic.Field(default='a', description='test')
f.from_field(default='b')
-> FieldInfo(annotation=NoneType, required=False, default='b')
```

I'm not sure the way this is supposed to be done, but there is a merge field info:
```py
FieldInfo.merge_field_infos(f, FieldInfo(default='b'))
-> FieldInfo(annotation=NoneType, required=False, default='b', description='test')
```

**Review by amitschang — COMMENTED** [review]
Some comments

File: evolver/hardware/standard/od_sensor.py (hunk 1 above)
Comment (jamienoss) [reply evolver/hardware/standard/od_sensor.py:44]: Nice catch, I hadn't noticed that, thanks!

Comment (jamienoss) [reply evolver/hardware/standard/od_sensor.py:44]: Rereading the [the docs for this](https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.FieldInfo.from_field), I really don't see the point of this method at all. Seems like quite a wasted opportunity and my, assumed functionality, would be far more useful. Never mind...

Comment (jamienoss) [reply evolver/hardware/standard/od_sensor.py]: ``merge_field_infos`` did the trick, thanks again.

**Review by amitschang — APPROVED** [review]
looks good, thanks!

**Pull request merged**